packages/vscode-extension/src/migration/migrationHandler.ts

```diff
@@ -151,6 +132,17 @@ async function updateCodeInplace(
 ): Promise<Result<null, FxError>> {
   try {
     const sourceCode = (await fs.readFile(filePath)).toString();
+    // skip files that do not import Teams client SDK
+    if (!sourceCode.includes(constants.teamsClientSDKName)) {
+      return ok(null);
+    }
+    vsCodeLogProvider.info(
```

Comment:
For the screenshot I see one INFO message and one WARNING message on the same file.
- [Info] Update xxx
- [Warning] Failed to update xxx

This could be confused. Consider remove the INFO one or change the word to *Updating*.
- [Info] Update xxx
- [Warning] Failed to update yyy
or
- [Info] Updating xxx
- [Info] Updating yyy
- [Warning] Failed to update yyy

Reply:
Good catch. Updated to use `[Info] Updating xxx`.